### PR TITLE
Fix bug related to pasting yaml in function

### PR DESCRIPTION
**Pull request by Lyczeq**
**Description**

Changes proposed in this pull request:

- Add getting dependencies and source code from yaml in the function

Close: #775
**Related issue(s)**

### CODE REVIEW

File: core-ui/src/components/Predefined/Create/Functions/Functions.create.js [hunk 1]
```diff
@@ -74,9 +75,13 @@ export function FunctionsCreate({
       jp.value(
         func,
         '$.spec.source',
-        CONFIG.defaultLambdaCodeAndDeps[runtime].code,
+        source || CONFIG.defaultLambdaCodeAndDeps[runtime].code,
```
Comment (mrCherry97) [inline core-ui/src/components/Predefined/Create/Functions/Functions.create.js:78]: Do we need that `||`? Without it it's working well.
```
      jp.value(func, '$.spec.source', source);
      jp.value(func, '$.spec.deps', deps);
```

File: core-ui/src/components/Predefined/Create/Functions/Functions.create.js [hunk 2]
```diff
@@ -74,9 +75,13 @@ export function FunctionsCreate({
...
+      );
+      jp.value(
+        func,
+        '$.spec.deps',
+        deps || getDefaultDependencies(name, runtime),
```
Comment (mrCherry97) [inline core-ui/src/components/Predefined/Create/Functions/Functions.create.js:83]: Do we need that `||`? Without it it's working well.
```
      jp.value(func, '$.spec.source', source);
      jp.value(func, '$.spec.deps', deps);
```

**Review by mrCherry97 — COMMENTED** [review]
Just these questions and LGTM for me :D

**Review by mrCherry97 — APPROVED** [review]

**Review by dariadomagala — APPROVED** [review]

**Review by dariadomagala — APPROVED** [review]

**Pull request merged**